Infomaniak Public Cloud 01 (S3).cyberduckprofile

```diff
@@ -37,16 +37,8 @@
         <false/>
         <key>Anonymous Configurable</key>
         <false/>
-        <key>Region</key>
-        <string>auto</string>
-        <key>Regions</key>
-        <array>
-            <string>auto</string>
-        </array>
         <key>Properties</key>
         <array>
-            <string>s3.storage.class.options=STANDARD</string>

```

Comment:
There is no need to remove this explicit configuration although the default [^1] is now the same.

[^1]: https://github.com/iterate-ch/cyberduck/blob/ffc47d9fcad1cd16b74724064cd2d9f99d1141d2/defaults/src/main/resources/default.properties#L311

Reply:
I've add the option in all profile